ably/rest_channel_test.go

```diff
@@ -187,4 +196,299 @@ func TestRestChannel(t *testing.T) {
 			}
 		}
 	})
+
+}
+
+func TestIdempotentPublishing(t *testing.T) {
+	t.Parallel()
+	app, err := ablytest.NewSandboxWIthEnv(nil, "idempotent-dev")
```

Comment:
Once this functionality is integrated we will be removing any special-casing of this environment. Instead of having the custom environment specified in more than one place it would be best please if there can be a single constant or switch we can change to restore the default.